util/fibers/listener_interface.cc

```diff
@@ -204,6 +214,7 @@ void ListenerInterface::RunSingleConnection(Connection* conn) {
   clist->Unlink(conn);
 
   guard.reset();
+  open_connections_.fetch_sub(1, std::memory_order_relaxed);
```

Comment:
I suggest changing it to `memory_order_release`. Otherwise, nothing prevents from compiler/CPU  to move this statement up.